src/components/User.vue

```diff
@@ -29,6 +29,8 @@ export default {
     name() {
       if (this.profile?.name) {
         return this.profile.name;
+      } else if (this.profile && this.profile.ens) {
```

Comment:
You can use optional chaining here
```suggestion
      } else if (this.profile?.ens) {
```